src/js/main.js

```diff
@@ -274,17 +274,18 @@ Site.Countdown = {
 };
 
 Site.Modules = {
+  played: false,
   init: function() {
     var _this = this;
 
     _this.$videos = $('.module-video');
 
-    // TODO: Launch enter animation
+    _this.threshold = _this.$videos.offset().top - ($(window).height() / 3) * 2;
```

Comment:
This doesn't handle a resized window. But is a solution to the splash video trigger I think.

Reply:
True, this needs to respond to resize.